server/migrations/20190411042815-create-bookmark.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Bookmarks', {
+      id: {
+        type: Sequelize.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4,
+        unique: true,
+        allowNull: false
+      },
+      articleId: {
+        type: Sequelize.UUID
+      },
+      userId: {
+        type: Sequelize.UUID
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.fn('now')
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.fn('now')
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('Bookmarks');
+  }
+};
```

Comment:
@samorahno  EOF rules, add an end of a line

Reply:
This will be fixed @davidokonji